Doc/reference/datamodel.rst

```diff
@@ -1578,8 +1578,8 @@ Class Binding
    ``A.__dict__['x'].__get__(None, A)``.
 
 Super Binding
-   If ``a`` is an instance of :class:`super`, then the binding ``super(B,
-   obj).m()`` searches ``obj.__class__.__mro__`` for the base class ``A``
+   If ``a`` is an instance of :class:`super`, then the binding ``super().m()``
```

Comment:
I've restored the 'super binding' section to its previous form.  I didn't add any other lines based on your comment that the whole description may need clarification.